configs/config-tfm.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * \file config-tfm.h
+ *
+ * \brief TF-M configuration with tweaks for a successful build and test.
```

Comment:
The description is not very clear.

```suggestion
 * \brief TF-M "medium profile" configuration adapted to work on other platforms
```

Reply:
rather than "on other platforms", how about "standalone"?

Reply:
The adapted configuration needs a platform. But it needs a platform that Mbed TLS supports natively, as opposed to the TF-M configuration which specifically needs TF-M as a platform.